suitcase/utils/tests/conftest.py

```diff
@@ -49,7 +49,8 @@ def _internal_plan(dets):
     yield from _internal_plan(dets)
 
 
-@pytest.fixture(params=['det', 'direct_img', 'det direct_img'],
+@pytest.fixture(params=['det', 'direct_img', 'direct_img_list',
+                        'det direct_img'],
```

Comment:
```suggestion
                        'det direct_img direct_img_list'],
```

Reply:
perhaps we should also add the new detector to the 'group' pass for the detectors